### PR TITLE
CMA 2.12.1-394 release notes

### PR DESCRIPTION
**Pull request by mburke5678**
Errata: https://errata.devel.redhat.com/docs/show/131744

Peer review: [Custom Metrics Autoscaler Operator 2.12.1-394 release notes](https://76042--ocpdocs-pr.netlify.app/openshift-dedicated/latest/nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.html#nodes-pods-autoscaling-custom-rn-2121-394_nodes-cma-autoscaling-custom-rn) 

**Planned GA 5/20/24**

### CODE REVIEW

File: nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past.adoc [hunk 1]
```diff
@@ -10,6 +10,21 @@ The following release notes are for previous versions of the Custom Metrics Auto
 
 For the current version, see xref:../../../nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc#nodes-cma-autoscaling-custom-rn[Custom Metrics Autoscaler Operator release notes].
 
+[id="nodes-pods-autoscaling-custom-rn-2121-384_{context}"]
```
Comment (mburke5678) [inline nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past.adoc:13]: **PEER REVIEWER** The following text is unchanged, only moved from the current release notes (nodes-cma-autoscaling-custom-rn.adoc) to this file.

**Comment by mburke5678** [comment]
@joelsmith PTAL

**Comment by joelsmith** [comment]
LGTM, thanks!

**Review by kcarmichael08 — COMMENTED** [review]
LGTM!

**Pull request merged**